Products/Zuul/routers/manufacturers.py

```diff
@@ -51,8 +51,8 @@ def editProduct(self, params=None):
         """
         Edit a product
         """
-        oldData = self.getProductData(params['uid'], params['oldname'])
         facade = self._getFacade()
+        oldData = facade.getProductData(params['uid'], params['oldname'])[0]
```

Comment:
I don't understand this change. Is it because you want to avoid marshalling data inside `self.getProductData`)?